pages/editor.vue

```diff
@@ -85,6 +86,23 @@
         });
     }
 
+    function handleGoingBack() {
+        if (userDidSomething.value) {
+            goingBackPopUpShow.value = true;
+        } else {
+            return reloadNuxtApp({
+                path: '/',
+                ttl: 1000,
+            });
```

Comment:
Navigate to leave the change of the pages active. So if you then continue from the Index, you will build on what is already there instead of starting new.

Reply:
yeah I forgot about it. It was not in the initial task but when talking with Florent we thought it would need to be there as well. I added it to the task but not to the code. Should be good now.